### ISSUE TITLE
Create a `basename` option for the prompt's working_dir segment

### ISSUE THREAD

**Issue by itshaydendev** [petobens/trueline#21]
Then instead of `~ > Workspace -> git-repo` we could just have `git-repo`.

Perhaps allow just having this when you're in a git repo? So it's relative to the repo root?

**Comment by itshaydendev** [comment]
(with regards to the git repo one: `git rev-parse --show-toplevel` will get the absolute path to the root of the current repository)

**Comment by petobens** [comment]
At the moment you can set
```sh
TRUELINE_WORKING_DIR_ABBREVIATE_PARENT_DIRS=true
TRUELINE_WORKING_DIR_ABBREVIATE_PARENT_DIRS_LENGTH=1
```
to get `~ > W > git-repo`. I think that having some indication of the full path (even as minimal as a single letter) is generally useful.

**Comment by itshaydendev** [comment]
Yes, it is, but it would also be useful (for smaller terminals open alongside an editor on a laptop, for instance) to have a basename-based working_dir as well

**Comment by petobens** [comment]
Ok. I might consider adding it. Feel free  to send a PR in the meantime.

**Comment by itshaydendev** [comment]
If I get the chance, I will. My Bash skills aren't that good though, haha
